### ISSUE TITLE
Make Encrypt button disabled darker

### ISSUE THREAD

**Issue by vulopez** [vulopez/encryptor#17]
# Descrition
Use a darker gray for disabled button.

- To be it more consistent.

**Comment by vulopez** [comment]
👀 Should be used `--gray-200` in all disabled element.
